### DISCUSSION TITLE
怎么做统计？

### DISCUSSION THREAD

**Discussion by jxjxjhjh** [obsidianzh/forum#60] [Q&A常见问题]
记日记的时候，有一项是记录每天使用的气球数量，要怎么做统计？
比如我今天的日记中写了使用气球一个（关键词 气球），明天也写了 气球 关键词，要怎么统计总数？

还是记日记，比如今天记录了使用气球的体验如何，明天也记录了，要怎么自动把这一段体验汇集到一个文件中（或者在软件中可以直接查看所有相关内容的方式（我能想到的是搜索关键词的方式，但是感觉不太好）

**Comment by pyrochlore** [comment]
obsidian-tracker 可以帮助您，请参考 [obsidian-tracker](https://github.com/pyrochlore/obsidian-tracker)
